bin/cron/create_rollup_tables

```diff
@@ -113,6 +105,12 @@ def main
         }
       end
     end
+  rescue ActiveRecord::RecordNotFound => exception
+    Honeybadger.notify(
+      exception,
+      error_message: "Error reading level for rollup tables."
+    )
+    next
```

Comment:
I think it would be helpful to include the parent level name and contained level name in the error message.